ui/lib/replication/addon/templates/components/path-filter-config-list.hbs

```diff
@@ -0,0 +1,125 @@
+<h4 class="title is-5">
+  Filtered paths
+</h4>
+<div class="radio-card-selector">
+  <label
+    for="no-filtering"
+    class="radio-card
+      {{if (not config.mode) " is-selected"}}"
+  >
+    <div class="radio-card-row">
```

Comment:
out of curiosity, why aren't the radio cards a separate component? seems like they could be reused here (and i'm also selfishly asking because we are going to need very similar markup for the cards used in the tools and transit redesigns) 😉 

either way, i like the way you gave them their own component css file since that'll make component-izing them easier down the line. 😄

Reply:
I wasn't sure if this would be flexible for other places yet and if it was planned to use other places, but thought it might be. I think replication enable might move to these, but wasn't sure if it should be flexible enough to also work for where we enable secret engines and auth methods. So I made it easy to pull out, but left the component-y decisions for when you know where it'll be used.